Tools/MiniBrowser/MiniBrowser.xcodeproj/project.pbxproj

```diff
@@ -204,6 +246,7 @@
 			projectRoot = "";
 			targets = (
 				8D1107260486CEB800E47090 /* MiniBrowser */,
+				DD403C872851B13F00D899FC /* MiniBrowser (Mac-only) */,
```

Comment:
The naming of the targets is confusing, but I do not have a better suggestion.

Reply:
I've changed the aggregate target's name to `MiniBrowser (Platform filters)`, which hopefully helps to clarify its purpose.